### PR TITLE
CVE-2011-1811, CVE-2011-2836, and CVE-2010-0656

### PR DESCRIPTION
**Pull request by rxh4133**
CVE-2011-1811
CVE-2011-2836
CVE-2010-0656

Ryan Hartzell

### CODE REVIEW

File: cves/CVE-2011-1811.yml [hunk 1]
```diff
@@ -55,9 +56,11 @@ unit_tested:
```
Comment (Norwal) [inline cves/CVE-2011-1811.yml]: This gets 3 upvotes

File: cves/CVE-2011-2836.yml [hunk 2]
```diff
@@ -52,9 +54,11 @@ unit_tested:
```
Comment (Norwal) [inline cves/CVE-2011-2836.yml]: This gets 4 of my upvotes

File: cves/CVE-2011-2836.yml [hunk 3]
```diff
@@ -52,9 +54,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    Automated unit tests were not involved in this vulnerability.
+    It was not unit tested, and the fix did not involve imporving the tests.
```
Comment (Norwal) [inline cves/CVE-2011-2836.yml]: Improving instead of imporving

File: cves/CVE-2010-0656.yml [hunk 4]
```diff
@@ -112,8 +113,8 @@ major_events:
     we want to capture what the development team was dealing with at the time.
   answer: 
   events:
-  - name: 
-    date: 
+  - name: The fixes break some unit tests
+    date: 2009-11-22
```
Comment (Norwal) [inline cves/CVE-2010-0656.yml]: If the fixes broke some unit tests, was the fixing commit reverted?

File: cves/CVE-2011-1811.yml [hunk 5]
```diff
@@ -55,9 +56,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    Automated unit tests were not involved in this vulnerability.
+    It was not unit tested, and the fix did not involve imporving the tests.
```
Comment (Norwal) [inline cves/CVE-2011-1811.yml]: Improving instead of imporving

File: cves/CVE-2011-2836.yml [hunk 6]
```diff
@@ -170,4 +175,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    They knew that the windows media player plugin was a backdoor for a while, and didn't solve it.
+    The bug was open for over a month before someone took ownership and started to fix it.
```
Comment (Norwal) [inline cves/CVE-2011-2836.yml]: Was there any reason given on why no one took ownership of this bug?

File: cves/CVE-2010-0656.yml (hunk 4 above)
Comment (tommydvelopment) [inline cves/CVE-2010-0656.yml]: I noticed that in the files you changed in this commit, you did not include any CWE's. They were a pain for me to find at first but if you look up your CVE on nvd.nist.gov/ they should all be listed there.

File: cves/CVE-2011-1811.yml (hunk 1 above)
Comment (tommydvelopment) [inline cves/CVE-2011-1811.yml]: I award this 5 of my upvotes.

Comment (tommydvelopment) [inline cves/CVE-2011-1811.yml]: And I will award this 5 of my upvotes.

File: cves/CVE-2010-0656.yml [hunk 7]
```diff
@@ -15,7 +15,8 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
```
Comment (tommydvelopment) [inline cves/CVE-2010-0656.yml]: I noticed that in the files you changed in this commit, you did not include any CWE's. They were a pain for me to find at first but if you look up your CVE on nvd.nist.gov/ they should all be listed there.

File: cves/CVE-2011-1811.yml [hunk 8]
```diff
@@ -173,4 +177,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    The original programmers did not think about the extreme cases of their code.
+    They didn't put a obvious limit on chrome form autofills.
```
Comment (tommydvelopment) [inline cves/CVE-2011-1811.yml]: What was the finite limit they ended up setting?

**Pull request merged**